cluster/examples/kubernetes/ceph/csi/rbd/storageclass.yaml

```diff
@@ -7,6 +7,12 @@ spec:
   failureDomain: host
   replicated:
     size: 3
+    # gives a hint (%) to Ceph in terms of expected consumption of the total cluster capacity of a given pool
+    # for more info: https://docs.ceph.com/docs/master/rados/operations/placement-groups/#specifying-expected-pool-size
+    #targetSizeRatio: .5
+    # Allow setting pool with replica 1, this could lead to data loss without recovery.
+    # Make sure you're *ABSOLUTELY CERTAIN* that is what you want
+    #allowUnsafeSingleReplica: false
```

Comment:
How about if we just set this to false instead of commenting it out? Then the comment can mention the warning about changing it to true. Looks like some of the other yamls follow this pattern already.